lib/generator._js

```diff
@@ -264,10 +265,19 @@ ScriptGenerator.prototype.generateAspNetCoreDeploymentScript = function (_) {
     throw new Error('Only batch script files are supported for ASP.NET Core Application');
   }
 
+  if (!this.solutionPath && this.useMSBuild) {
+    throw new Error('Missing solution file path (--solutionFile). Solution file path has to be specified when used with --use-msbuild');
```

Comment:
If the caller doesn't pass `useMSBuild` but pass a solution file then the solution file will be ignored. Should we throw on that as well? I guess we can.

Though this remind me to update `--use-msbuild` to `--useMSBuild`